Traceback (most recent call last):
  File "/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/requests/compat.py", line 11, in <module>
    import chardet
ModuleNotFoundError: No module named 'chardet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/users/acp23jlc/miniproject/speechbrain/TEAM2_modifications/../recipes/LibriSpeech/ASR/transformer/TEAM2_train_with_whisper.py", line 21, in <module>
    import speechbrain as sb
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/__init__.py", line 4, in <module>
    from .core import Stage, Brain, create_experiment_directory, parse_arguments
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/core.py", line 38, in <module>
    from speechbrain.utils.optimizers import rm_vector_weight_decay
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/utils/__init__.py", line 11, in <module>
    from . import *  # noqa
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/utils/parameter_transfer.py", line 14, in <module>
    from speechbrain.pretrained.fetching import fetch, FetchFrom, FetchSource
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/pretrained/__init__.py", line 3, in <module>
    from .interfaces import *  # noqa
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/pretrained/interfaces.py", line 31, in <module>
    from speechbrain.pretrained.fetching import fetch
  File "/users/acp23jlc/miniproject/speechbrain/speechbrain/pretrained/fetching.py", line 16, in <module>
    from requests.exceptions import HTTPError
  File "/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/requests/__init__.py", line 45, in <module>
    from .exceptions import RequestsDependencyWarning
  File "/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/requests/exceptions.py", line 9, in <module>
    from .compat import JSONDecodeError as CompatJSONDecodeError
  File "/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/requests/compat.py", line 13, in <module>
    import charset_normalizer as chardet
  File "/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/charset_normalizer/__init__.py", line 23, in <module>
    from charset_normalizer.api import from_fp, from_path, from_bytes, normalize
  File "/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/charset_normalizer/api.py", line 10, in <module>
    from charset_normalizer.md import mess_ratio
  File "charset_normalizer/md.py", line 5, in <module>
ImportError: cannot import name 'COMMON_SAFE_ASCII_CHARACTERS' from 'charset_normalizer.constant' (/users/acp23jlc/.conda/envs/speechbrain_venv/lib/python3.9/site-packages/charset_normalizer/constant.py)
